modules/diskq/qdisk.c

```diff
@@ -1388,6 +1387,32 @@ _autodetect_disk_buf_size(QDisk *self)
   return FALSE;
 }
 
+static gboolean
+_handle_option_changes(QDisk *self)
```

Comment:
I renamed the function and reordered it a bit:
```c
static gboolean
_ensure_disk_buf_size(QDisk *self)
{
  if (self->hdr->disk_buf_size == -1 && !_autodetect_disk_buf_size(self))
    return FALSE;

  if (self->hdr->disk_buf_size != self->options->disk_buf_size)
    {
      msg_warning("WARNING: disk-buf-size() has changed since the last syslog-ng run. syslog-ng currently does "
                  "not support changing the disk-buf-size() of existing disk-queues. Continuing with the old one",
                  evt_tag_str("filename", self->filename),
                  evt_tag_long("active_old_disk_buf_size", self->hdr->disk_buf_size),
                  evt_tag_long("ignored_new_disk_buf_size", self->options->disk_buf_size));
    }

  return TRUE;
}
```

Reply:
Thanks.